src/Tablerow.js

```diff
@@ -0,0 +1,43 @@
+import React, { useState } from "react";
+import Moment from "moment";
+
+function CreateTableRow({ profile, index, showProfile }) {
+  const [colour, setColor] = useState(false);
+  function handleClick() {
+    setColor(oldColour => !oldColour);
+  }
```

Comment:
you already have the colour variable in state, so you can just have this as `setColour(!colour)`, that way it sets the colour boolean to the opposite of what is currently in state